README.md

```diff
@@ -0,0 +1,179 @@
+
+If you prefer to edit raw configuration JSON, make sure inside the main block your `modules` config looks something like this:
+
+``` json
+{
+  "modules": [
+    {
+      "executable_path": "/home/username/periph_board/periph_board",
+      "name": "periph"
+    }
+  ],
+  "components": []
+}
+```
+
+### Configure component
+
+After configuring the module, configure the component as type *board*, model *viamlabs:board:periph*.
+You may choose any `Name` you want.
+
+If you prefer to edit raw configuration JSON, make sure inside the main block your `components` config looks something like this:
+
+``` json
+{
+  "modules": [
+    {
+      "executable_path": "/home/username/periph_board/periph_board",
+      "name": "periph"
+    }
+  ],
+  "components": [
+    {
+      "name": "local",
+      "type": "board",
+      "model": "viamlabs:board:periph",
+      "attributes": {},
+      "depends_on": []
+    }
+  ]
+}
+```
+
+## viam-server board setup examples
+
+### Orange Pi Zero 2
+
+The [Orange Pi Zero 2](http://www.orangepi.org/html/hardWare/computerAndMicrocontrollers/details/Orange-Pi-Zero-2.html) is an open-source SBC with dual-band wireless WiFi and Bluetooth 5.0 .It is highly compact with a dimension of 53X60mm. It can run Android10, Ubuntu, Debian.
+
+Start by downloading the latest Linux distribution from OrangePi.
+We tested and had success with the [Debian bullseye distribution](https://drive.google.com/drive/folders/1Xk7b1jOMg-rftowFLExynLg0CyuQ7kCM).
+
+Flash this image to an SD card with [Balena Etcher](https://etcher.balena.io/)
+
+Then insert the flashed SD card to the Orange Pi, plug the Pi into a monitor with an mini-HDMI cable, and plug in the Pi to with a 5v power supply.
+Log in with the credentials root/orangepi.
+
+Secure the Pi:
+
+- Change the root password by issuing the `passwd` command.
+- Change the password for the `orangepi` user account by issuing the commmand `passwd orangepi`
+
+Update the software:
+
+``` bash
+apt update
+apt upgrade
+```
+
+Enable wifi:
+
+```bash
+nmcli dev wifi
+nmcli dev wifi connect <wifi_name> password <wifi_passwd>
+```
+
+Enable the host being announced on the network by running `orangepi-config` and going to System->Avahi and *Enable system announce in the network*.
+
+While still in `orangepi-config`, change the hostname to your preference by going to Personal->Hostname.
+
+Enable i2c and spi by editing `/boot/orangepiEnv.txt`, adding the line:
+
+``` bash
+overlays=i2c3 spi-spidev
+param_spidev_spi_bus=1
+param_spidev_spi_cs=1
+```
+
+Now install the Orange Pi gpio tool:
+
+``` bash
+git clone https://github.com/orangepi-xunlong/wiringOP
+cd wiringOP
+sudo ./build clean
+sudo ./build
+gpio readall
+```
+
+At this point, you can unplug the HDMI cable and restart.
+You should now be able to connect to the Pi headless via `ssh <user>@<hostname>.local` and [install viam-server](https://docs.viam.com/installation/#install-viam-server)
+
+### Rock Pi S
```

Comment:
Ohh, I think I get it now! This PR is a combination readme for the module, and setup guide for 2 different boards! What would you feel about making it 3 different files instead? The readme could link to the other two as example setups, but then it's clear which parts are about this component and which are specific to boards that the user might not even own.

Reply:
maybe later